docs/self-managed/setup/upgrade.md

```diff
@@ -5,12 +5,10 @@ sidebar_label: "Upgrade"
 description: "To upgrade to a more recent version of the Camunda Helm charts, there are certain things you need to keep in mind."
 ---
 
-To upgrade to a more recent version of the Camunda Helm charts, there are certain things you need to keep in mind.
-
-:::caution
-
-Ensure to review the [instructions for a specific version](#version-update-instructions) before starting the actual upgrade.
+To upgrade to a more recent version of the Camunda Helm charts, configuration adjustments may be required. Review the [instructions for specific versions](#version-update-instructions) prior to starting an upgrade.
 
+:::note
+The recommended upgrade path for any given installation is to the **latest patch** release of the **next minor version**.
```

Comment:
I'd like to see a review from the Distribution team, specifically for this note. Specifically because in the [update instructions](https://docs.camunda.io/docs/self-managed/operational-guides/update-guide/introduction/) we mention the following:

> When updating from one minor version to the next, you do not need to update to each patch version along the way. However, do not skip minor versions when updating.

Should we also mention (again), in this PR, that you should not skip minor versions? Or do we think this is clear enough as is?

Reply:
@hamza-m-masood, can we get your review here?

Reply:
Sorry @akeller , you'll just have to put up with a review from me :stuck_out_tongue_winking_eye: . 

I do not find the two statements:

(statement A)
> When updating from one minor version to the next, you do not need to update to each patch version along the way. However, do not skip minor versions when updating.

and

(statement B)
> The recommended upgrade path for any given installation is to the latest patch release of the next minor version.

to be contradictory.  I think the difference between them is that one states what not to do, and the other describes what to do. And I think both are clear that one minor upgrade at a time is the acceptable approach.

However, one thing that I'm realizing is that the "meaning" of a minor version is different now that we have the detached version numbers.  We should re-think whether this statement is intended for people thinking about the camunda version numbers or the helm chart version numbers.  if it's the helm chart version numbers, we need to say 

```suggestion
:::note
The recommended upgrade path for any given installation is to the **latest patch** release of the **next major version**.
```

Reply:
@jessesimpson36 appreciate you 🙂 

Great point about the detached version numbers. The only additional thing I would add to your suggestion is to qualify that this is for Helm charts.

@conceptualshark can you have a look at this feedback and adjust the PR?

Reply:
@jessesimpson36 @akeller The original request is [here](https://github.com/camunda/camunda-docs/issues/3857) - I was making the assumption it meant camunda versions, but documented on the Helm chart page. Would it be appropriate to update this note to include a specific recommendation for each?

Reply:
I'm good with saying this page refers to major versions because we are within the context of Helm. So, we recommend the latest patch release of the next major version (of the Helm chart). I would rather not go into more detail about each. If we get more feedback on this page, we can always adjust it.